webrender/src/platform/macos/font.rs

```diff
@@ -210,6 +219,24 @@ impl FontContext {
         }
     }
 
+    pub fn get_glyph_index(&mut self, font_key: FontKey, ch: char) -> Option<u32> {
+        let characters: [u16; 1] = [ ch as u16 ];
```

Comment:
is there a point of using fixed size arrays of size 1 here as opposed to straight values?